### DISCUSSION TITLE
server-mod-name features

### DISCUSSION THREAD

**Discussion by UsainSrht** [PurpurMC/Purpur#1062] [__Dismissed__] [answered]
Hex color support _if possible_
A method to change specific player's server-mod-name without needing a relog. _player.setServerModName(Component name)_

![Screenshot_1](https://user-images.githubusercontent.com/59025999/179607781-43a36c3c-176d-46db-9891-a0b678304aa9.png)
![Screenshot_2](https://user-images.githubusercontent.com/59025999/179607790-27e86b5a-4bd9-4407-a2ea-2dd8bbe7e104.png)

**Comment by UsainSrht** [comment]
I used spigot's hex color code format for below picture `&x&r&r&g&g&b&b`
and it showed last color which is 0

**Comment by katekerllenevich** [comment, marked as answer]
Unfortunately this isn't  a component. You only get the last color because the client still parses legacy client codes every now and then.

**Reply by UsainSrht** [reply]
Thanks, I didn't know that.
